fe/fe-core/src/main/java/com/starrocks/sql/analyzer/Analyzer.java

```diff
@@ -134,6 +135,12 @@ public Void visitDeleteStatement(DeleteStmt node, ConnectContext context) {
             DeleteAnalyzer.analyze(node, context);
             return null;
         }
+
+        @Override
+        public Void visitDropMaterializedViewStatement(DropMaterializedViewStmt statement, ConnectContext context) {
```

Comment:
can add enableNewMaterializedView here?
if. enableNewMaterializedView is false, throw an exception here?